src/main/java/build/buildfarm/cas/cfc/CASFileCache.java

```diff
@@ -981,6 +987,8 @@ public synchronized FeedbackOutputStream getOutput(
               throws IOException {
             // caller will be the exclusive owner of this write stream. all other requests
```

Comment:
misplaced comment

Reply:
Not my comment, but I can remove it 🤷‍♂️